tests/integration/targets/iosxr_interfaces/tasks/main.yaml

```diff
@@ -1,4 +1,4 @@
 ---
 - include: cli.yaml
   tags:
-    - network_cli
+    - iosxr_nonvoting
```

Comment:
Why is `iosxr_nonvoting` tag added here?

Reply:
Because this particular module is failing on CI with intermittent issue for delete (after scenario)